### PR TITLE
Enable some testing

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -391,6 +390,8 @@ testing_task:
         - "build_each_commit"
         - "build_without_cgo"
 
+    allow_failures: skip: $CI == 'true'
```
Comment (cevich) [inline .cirrus.yml:393]: `s/skip: //g`

**Comment by mheon** [comment]
LGTM

**Pull request merged**